### PR TITLE
React-Capstone 

### PR DESCRIPTION
**Pull request by HenaryKhuraijam**
## React-Capstone 
- Created a Component The Retrieves and Data from an External API.
- Created Another Component to Fetch Detail Information.
- Created action and reducer to fetch data from API and store it in the store.
- Written Unit Test for pure functions.
- [Live Demo Link](https://moviesworld-gn6u.onrender.com)
- [Link to video presentation](https://www.loom.com/share/1eabc83d58454ff1bb40a0794275e646?sid=dfe93218-dbfa-4beb-a1e4-900998b9313c)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,33 +143,34 @@ You can redeploy this project by adding new lines of code to source files.
...
 
-Give me ⭐️ If you like this project!
+If you like this project feel free to leave comments and contact via given authors information above.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
-
-- I would like to thank Microverse program for providing me this great chance.
+I would like to thank the code reviewers for taking time to do the work. A special thank you as well to my teammates for their support it wouldnt have been possible without you guys.
```
Comment (Alusp) [inline README.md]: - [ ] Good job so far, :clap: , However, according to the project requirement, it requires that you give appropriate credit to the author. Please resolve that.

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @HenaryKhuraijam, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
close

## Highlights

- [ ] Use Professional Readme file 👍
- [ ] Descriptive PR Summary used 👍
- [ ]  Use React components.👍
- [ ] Handle events in a React app. 👍
- [ ] Use store, actions, and reducers in React. 👍



## Required Changes ♻️
_Check the comments under the review._

- [ ] According to the project requirement please ensure that you highlight a specific piece of code in your video presentation you are most proud of. Please resolve that. :slightly_smiling_face: 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL BUT HIGHLY RECOMMENDED] Please ensure that your homepage logo is clickable and takes you to the homepage when clicked. below is the screenshot of the homepage icon. :arrow_down: 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alusp in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (HenaryKhuraijam) [reply README.md]: Done

**Review by Alusp — APPROVED** [review]
Hi @HenaryKhuraijam, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
congrats
## Highlights

- [ ] Use Professional Readme file 👍
- [ ] Descriptive PR Summary used 👍
- [ ]  Use React components.👍
- [ ] Handle events in a React app. 👍
- [ ] Use store, actions, and reducers in React. 👍

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL BUT HIGHLY RECOMMENDED] Please ensure that your homepage logo is properly aligned, clickable, and takes you to the homepage when clicked. below is the screenshot of the homepage icon. :arrow_down: 
image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alusp in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**